### DISCUSSION TITLE
Request steps and .YML example to create mask: 'image : mask02.bmp'

### DISCUSSION THREAD

**Discussion by deepcoder** [blakeblackshear/frigate#421] [Ask A Question]
Request steps and .YML example to create mask: 'image : mask02.bmp'

I have two disconnected areas that I do not want detection to include, so I am of the understanding that I need to go with an image mask.

Can someone share the steps to create the right format image mask file? I created a black and white .BMP file from a:
http://192.168.1.100:5000/cam-01/latest.jpg
image.

But I don't thing I have the image correct. Correct dimensions? Correct depth? Correct format? Where and how in the .YML file to put the configuration?

If someone can share a set of steps and configuration that is working for them, that would be appreciated. Thx!

**Comment by mattheys** [comment]
Dimensions should be the same as your camera, depth could be 1bit, format is BMP. Then you just put in the mask file in your config directory and the filename in the config file. 

https://github.com/blakeblackshear/frigate/blob/a1cc9ad1f0a3ea3e3a1c8163c5ff3c27065a9be4/README.md#L270-L273

However I would reccomend using a Poly mask and make a very thin connecting line between the two areas, you could make it 0px wide if you go straight across and tweak it by hand. I think v0.8 will have support for a list of poly masks which would be good for your circumstance.

https://github.com/blakeblackshear/frigate/blob/a1cc9ad1f0a3ea3e3a1c8163c5ff3c27065a9be4/README.md#L454-L485
